mealplanner-ui/src/pages/ShoppingList.tsx

```diff
@@ -37,10 +37,14 @@ const shoppingListQuery = graphql`
                 quantity
                 unit
                 productKeyword
+                substituteIngredient {
+                  name
```

Comment:
We can also get the reason for the substitute

Reply:
okay, I will add substitute reason